apsi-enrollment-be/src/main/java/edu/pw/apsienrollment/enrollment/EnrollmentService.java

```diff
@@ -0,0 +1,8 @@
+package edu.pw.apsienrollment.enrollment;
+
+import edu.pw.apsienrollment.enrollment.api.dto.EnrollmentRequestDto;
+import edu.pw.apsienrollment.enrollment.db.Enrollment;
+
+public interface EnrollmentService {
+    Enrollment signUp(EnrollmentRequestDto enrollmentRequestDto);
```

Comment:
nit. I would pass only the id to the service, not the entire dto to separate service and api layers

Reply:
Done